src/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import App from './theme/App';
```

Comment:
Use absolute imports please - Google how to configure absolute imports for VSCode
```suggestion
import App from 'src/theme/App';
```

Reply:
path updated.